clusters/ztp-policies/common/common-policies/common-ranGen-vse414.yaml

```diff
@@ -78,6 +78,3 @@ spec:
           - mirrors:
               - brew.registry.redhat.io/openshift-logging
             source: registry.redhat.io/openshift-logging
-          - mirrors:
-              - quay.io/redhat-partner-solutions/prega/intel
```

Comment:
Don't we need this for Pre GA operators?

Reply:
No, this is my catalog